data-processing-lib/python/src/data_processing/utils/transform_configuration.json

```diff
@@ -0,0 +1,247 @@
+{
```

Comment:
should we add automatic update, or verification of this file? E.g. if a new sub-directory is created in the transforms dir.

Reply:
why?